test/test_sparse.py

```diff
@@ -102,14 +103,14 @@ def test_print(self):
             printed.append("# indices shape: {}".format(indices_shape))
             printed.append("# values shape: {}".format(values_shape))
 
-            indices = torch.arange(indices_shape.numel(), dtype=self.IndexTensor.dtype,
+            indices = torch.arange(indices_shape.numel(), dtype=self.index_tensor(0).dtype,
                                    device=self.device).view(indices_shape)
             for d in range(sparse_dim):
                 indices[d].clamp_(max=(shape[d] - 1))  # make it valid index
             if self.is_uncoalesced and indices.numel() > 0:
                 indices[:, -1] = indices[:, 0]  # make it uncoalesced
             values_numel = values_shape.numel()
-            values = torch.arange(values_numel, dtype=self.ValueTensor.dtype,
+            values = torch.arange(values_numel, dtype=self.value_tensor(0).dtype,
```

Comment:
`value_dtype` and this one there's no excuses :)

Reply:
fair enough.